cmd/kontrastd/collector.go

```diff
@@ -40,14 +46,21 @@ func (c *KontrastCollector) Describe(ch chan<- *prometheus.Desc) {
 // metrics. The implementation sends each collected metric via the
 // provided channel and returns once the last metric has been sent.
 func (c *KontrastCollector) Collect(ch chan<- prometheus.Metric) {
+	resources := map[labelSet]float64{}
 	for _, file := range c.manager.GetDiffFiles() {
 		for _, resource := range file.Resources {
 			if resource.DiffResult.Status == DiffPresent {
-				objectLabel := fmt.Sprintf("%s/%s", resource.Kind, resource.Name)
-				ch <- prometheus.MustNewConstMetric(currentDiffsGauge,
-					prometheus.GaugeValue, 1.0,
-					objectLabel, resource.Namespace)
+				ls := labelSet{resource.Kind, resource.Name, resource.Namespace}
+				resources[ls] = resources[ls] + 1
 			}
 		}
 	}
+
+	for resource, _ := range resources {
```

Comment:
This is non-deterministic but I don't think it really matters

Reply:
Correct :+1: